client/landing/stepper/declarative-flow/internals/steps-repository/design-choices/index.tsx

```diff
@@ -36,9 +31,6 @@ const DesignChoicesStep: Step = ( { navigation, flow, stepName } ) => {
 	);
 
 	const { isEligible, isLoading } = useIsBigSkyEligible();
-
-	const isSiteAssemblerEnabled = useIsSiteAssemblerEnabled();
```

Comment:
We can now clean up `useIsSiteAssemblerEnabled()` definition (`client/data/site-assembler/index.ts`)

Reply:
Good catch 🙂